### PR TITLE
BaSP-MR-114: activities redux

### PR DESCRIPTION
**Pull request by LucianoStradiot**
Implement redux on activities entity

### CODE REVIEW

File: src/Components/Activities/FormActivity/index.js [hunk 1]
```diff
@@ -7,25 +7,31 @@ import Select from '../../Shared/Select';
...
-  const [activities, setActivities] = useState([]);
   const [modalInfo, setModalInfo] = useState({
     title: '',
     desc: ''
   });
   const [isOpen, setIsOpen] = useState(false);
   const [buttonAddIsVisible, setAddVisible] = useState(false);
   const [buttonSaveIsVisible, setSaveVisible] = useState(false);
-  const getActivities = async () => {
+  /* const getActivities = async () => {
```
Comment (Tilajc) [inline src/Components/Activities/FormActivity/index.js]: you can delete the commented lines
```suggestion

```

File: src/Redux/Activities/reducer.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+      return { ...state, isLoading: false, error: action.payload };
+    }
+    case DEL_ACTIVITIES_LOADING: {
+      return { ...state, isLoading: true, error: null };
+    }
+    case DEL_ACTIVITIES_SUCCESS: {
+      return {
+        ...state,
+        isLoading: false,
+        data: state.data.filter((activity) => activity._id !== action.payload.id),
```
Comment (Tilajc) [inline src/Redux/Activities/reducer.js]:
```suggestion
        data: state.data.filter((activity) => activity._id !== action.payload._id),
```

**Review by Tilajc — CHANGES_REQUESTED** [review]
I leave some comments, good job!

**Review by Tilajc — APPROVED** [review]
Good job!

**Comment by LucianoStradiot** [comment]
[Implement Redux on the Activities entity](https://trello.com/c/koE2LRu6/114-implement-redux-on-the-activities-entity)

**Comment by Katairala** [comment]
Tested locally on https://joaco-megarocket-app.vercel.app/
Test Results: Passed 💯 

It great! Nice work Luciano

**Pull request merged**